Dockerfile

```diff
@@ -13,14 +13,11 @@ RUN pip install -r requirements-dev.txt
 WORKDIR /build/
 ADD . /build/
 
-# The following command replaces the @VERSION@ string in setup.py and package.json
+# The following command replaces the version string in setup.py and package.json
 # with the tagged version number from GIT_TAG or `0.0.0` if GIT_TAG is not set
 ARG VERSION=${GIT_TAG:-0.0.0}
-RUN echo "Version = $VERSION"
-# Update line 6 in setup.py
-RUN sed -i "6 s/0.0.0/$VERSION/" setup.py
-# Update line 3 in package.json
-RUN sed -i "3 s/0.0.0/$VERSION/" package.json
+RUN sed -i "s+version=\'0.0.0\'+version=\'$VERSION\'+" setup.py
+RUN sed -i "s+\"version\": \"0.0.0\"+\"version\": \"$VERSION\"+" package.json
```

Comment:
There's no need to escape `\'` inside `""`.  Pre-existing, the `.` are regex and probably should be escaped.  Out of curiosity, why change from conventional `/` to `+`?
```
RUN sed -i "s/version='0\.0\.0'/version='$VERSION'/" setup.py
RUN sed -i "s/\"version\": \"0\.0\.0\"/\"version\": \"$VERSION\"/" package.json
```
Well, I probably would have just kept the existing regexes and just removed the line number address (and possibly fixed the `.` escaping) since it seems unlikely that will match the wrong thing.